features/FxA-98-improve-fraud-detection/README.md

```diff
@@ -0,0 +1,122 @@
+- Feature Name: Improve fraud detection
+- Start Date: 2016-05-25
+- Feature PR:
+- Feature Milestone:
+
+# Summary
+[summary]: #summary
+
+# Motivation
+[motivation]: #motivation
+
+We wish to increase security for FxA by improving the fraud detection the customs server can provide. We want a /v2 API that is more logically structured, and take the opportunity to do the same to the internal server code.
+
+We wish to utilize a more permanent data store so we can better detech fraud.
+
+The customs server v2 will lend itself to be more easily configurable and manageable (ie, Adding custom rules, manually block). Ops would like to be able to adapt during a prolonged attack, and have configuration update without rebuilding the server.
+
+## What Do We Gain
+
+- **detect access from new IP addresses**: The customs server will save IP address/email combinations, so that it can detect when a successful login attempt is from a "new" IP. The auth-server confirmation emails can be reduced to only send to users if login occurs from a "new" IP address.
```

Comment:
Having given a long thought, it may make sense to separate this part from this feature, and do as you suggested: add a `logins` table to fxa-auth-server, and do this check there.

This leaves the customs server to only calculate scores, where as `db.hasLoginBefore(req.ipAddr)` wouldn't ever give a score, other then "NEVER SEEN BEFORE! RED ALERT! SEND CONFIRMATION".